src/db.py

```diff
@@ -217,3 +218,6 @@ def get_extensions(cid, aid, netid=None):
 
 def add_extension(cid, aid, netid, max_runs, start, end):
 	return mongo.db.extensions.insert_one({"course_id": cid, "assignment_id": aid, "netid": netid, "max_runs": max_runs, "remaining_runs": max_runs, "start": start, "end": end})
+
+def delete_extension(extension_id):
+	return mongo.db.extensions.delete_one({"_id": ObjectId(extension_id)})
```

Comment:
Maybe do a try catch here instead of the level above. I think the `ObjectId` throws an exception when the id is invalid.